build_deps/gpus/check_cuda_libs.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Please change the header to the NVIDIA version referring to the other `cuh` files.